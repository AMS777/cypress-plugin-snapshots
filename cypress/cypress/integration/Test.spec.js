describe('Test', () => {
  it('toMatchSnapshot', () => {
    cy.request('/static/stub.json')
      .its('body')
      .toMatchSnapshot();
  });
});
