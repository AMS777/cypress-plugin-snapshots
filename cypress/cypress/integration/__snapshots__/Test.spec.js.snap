{
  "Test > toMatchSnapshot #0": {
    "number": 1234,
    "object": {
      "foo": "bar",
      "nestedObject": {
        "bar": "foo"
      }
    },
    "string": "string"
  }
}
